ReleaseNotes.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.5.0
+Post-training Quantization:
+- Features:
+  - Added Post-training Quantization support for OpenVINO IR (OpenVINO backend).
+  - Added `"overflow_fix"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support & functionality. It improves accuracy for optimized model for affected devices.
+  - Added `"model_type"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support that allows reach better accuracy for specific model architectures (e.g. transformers).
+  - Added support for in-place statistics collection (reduce memory footprint during optimization).
+  - Added `nncf.parameters.IgnoredScope` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support that allows exclude layers from optimization scope in different ways.
+  - (OpenVINO) Added Quantization with accuracy control algorithm.
+  - (OpenVINO) Added weights compression after quantization (reduced .bin size of the optimized model).
+  - (OpenVINO) Added support of the Depthwise & GroupConvolution layers support for correct quantization.
+  - (OpenVINO) Added YOLOv8 examples for [`quantize(...)`](examples/post_training_quantization/openvino/yolov8) & [`quantize_with_accuracy_control(...)`](examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control) methods.
+  - (OpenVINO) Added support for dynamic-shape models.
```

Comment:
Do you have prove for this statement?

Reply:
It depends on the validation scope. Do we have dynamic shape models? I thought yes, because we were dealing with issues related to them.